src/Crypto/crypto-pairings.asd

```diff
@@ -26,37 +26,14 @@ THE SOFTWARE.
   :description "crypto-pairings: bilinear pairings (PBC) functions"
   :version     "1.1.0"
   :author      "D.McClain <dbm@refined-audiometrics.com>"
-  :in-order-to ((test-op (test-op "crypto-pairings-test")))
+  :in-order-to ((test-op (test-op "crypto-pairings-tests")))
```

Comment:
Please use the singular for for packages and ASDF systems to be consistent:

"crypto-pairings-tests" --> ""crypto-pairings-test"

Reply:
yes, I thought I did that yesterday. I like your consistent convention. Looks like I need to go back and double check what I thought I did already.

Reply:
fixed